benchadapt/python/benchadapt/_machine_info.py

```
@@ -364,4 +388,15 @@ def _has_missing(info, mapping):
 
 def _exec_command(command):
     result = subprocess.run(command, capture_output=True)
+
+    # Some of these failures are expected, i.e. the command not executing isn't
+    # supposed to be fatal to this program here. However, it makes sense to log
+    # information about how exactly the child process invocation failed.
+    if result.returncode != 0:
+        log.info(
+            "child process returned with code %s, stderr prefix:\n %s",
+            result.returncode,
+            result.stderr.decode("utf-8").strip()[:500],
+        )
```

dang, swallowing errors is never nice to the peeps around you :)